### ISSUE TITLE
Can your code generate high resolution image from two low resolution images

### ISSUE THREAD

**Issue by LZDSJTU** [teammcr192/super-resolution#1]
Hello~
If I take two images from two cameras, and I want to generate an image with higher resolution. Can I use your code?
If it can, what should I run after compiling?
Thank you!

**Comment by jodyngo** [comment]
Yes, I got the same question here. I would like to test with multiple frames.
I tried testing with below command:

bin/SuperResolution --data_path=home/thungo/Downloads/super-resolution/test_data/text\

however, it has shown error as below

I0419 22:47:54.689354 25723 util.cpp:35] Running with OpenCV version 4.2.0-dev.
F0419 22:47:54.689486 25723 data_loader.cpp:48] Check failed: stat(path.c_str(), &path_stat) == 0 The given file or directory path 'home/thungo/Downloads/super-resolution/test_data/text' cannot be opened.

May you please give me an example command line for Super Resolution ?
Thank you

**Comment by jodyngo** [comment]
> Yes, I got the same question here. I would like to test with multiple frames.
> I tried testing with below command:
> 
> bin/SuperResolution --data_path=home/thungo/Downloads/super-resolution/test_data/text\
> 
> however, it has shown error as below
> 
> I0419 22:47:54.689354 25723 util.cpp:35] Running with OpenCV version 4.2.0-dev.
> F0419 22:47:54.689486 25723 data_loader.cpp:48] Check failed: stat(path.c_str(), &path_stat) == 0 The given file or directory path 'home/thungo/Downloads/super-resolution/test_data/text' cannot be opened.
> 
> May you please give me an example command line for Super Resolution ?
> Thank you

I resolved my problems. I missed a slash "/" before the folder path. The correct command line is

./SuperResolution  --data_path="/home/thungo/Downloads/super-resolution/test_data/text"

I hope it helps with your issue.
